libs/datamodel/parser-database/src/context.rs

```diff
@@ -296,7 +296,7 @@ impl<'db> Context<'db> {
             let attribute = &self.ast[*attribute_id];
             diagnostics.push_error(DatamodelError::new_attribute_not_known_error(
                 &attribute.name.name,
-                attribute.name.span,
+                attribute.span,
```

Comment:
@janpio this is the `@ignore` change. Now I remember: I think it's better if we highlight the whole attribute here, instead of just the name, because it's the attribute that isn't supported.

Reply:
I fundamentally agree, just confused me a bit for a minute. I think is fine to slip in here as well.

Reply:
yes, ideally that should be its own commit, it's an unrelated change